cdk/emr-serverless-with-sfn/stacks/sfn.py

```diff
@@ -0,0 +1,199 @@
+from typing import cast
+
+import aws_cdk as cdk
+from aws_cdk import Aws
+from aws_cdk import Stack as Stack
+from aws_cdk import aws_ec2 as ec2
+from aws_cdk import aws_iam as iam
+from aws_cdk import aws_s3 as s3
+from aws_cdk import aws_s3_deployment as s3deploy
+from aws_cdk import aws_stepfunctions as sfn
+from constructs import Construct, IConstruct
+
+from stacks.emr_serverless_sm import EmrServerlessStateMachineConstruct
+
+
+class SfnEmrServerlessJobsStack(Stack):
+    vpc: ec2.Vpc
+
+    def __init__(
+        self,
+        scope: Construct,
+        construct_id: str,
+        emr_serverless_app_id: str,
+        emr_serverless_app_arn: str,
+        bucket: s3.Bucket,
+        namespace: str,
+        **kwargs,
+    ) -> None:
+        """This stack creates a simple EMR Serverless demo within Step Functions.
+
+        We create a state machine that submits... TODO continue description
```

Comment:
Finish TODO

Reply:
oops sorry, totally missed that, thanks!